packages/headless/src/controllers/facets/range-facet/date-facet/date-range.test.ts

```diff
@@ -1,3 +1,5 @@
+import dayjs from 'dayjs';
```

Comment:
The changes in this file seems unrelated. 
What's the story behind those ?

Reply:
Since all those tests were done with useLocalTime = false, the values were all UTC. This causes a problem with hardcoding expected values, so I used dayjs everywhere

Reply:
Ah, makes sense. Thanks